tools/pd-ctl/README.md

```diff
@@ -197,7 +197,7 @@ Usage:
     >> config set split-merge-interval 24h  // Set the interval between `split` and `merge` to one day
     ```
 
-- `enable-one-way-merge` controls the merge scheduler behavior. This means a Region can only be merged into left.
+- `enable-one-way-merge` controls the merge scheduler behavior. This means a Region can only be merged into right.
```

Comment:
I think either `left` or `right` is ambiguous... How about `can only be merged into next Region of it`?

Reply:
done